packages/compat/src/compat-app.ts

```diff
@@ -225,21 +225,8 @@ export default class CompatApp {
 
   @Memoize()
   babelConfig(): TransformOptions {
-    // this finds all the built-in babel configuration that comes with ember-cli-babel
-    const babelAddon = (this.legacyEmberAppInstance.project as any).findAddonByName('ember-cli-babel');
-    const babelConfig = babelAddon.buildBabelOptions({
-      'ember-cli-babel': {
-        ...this.legacyEmberAppInstance.options['ember-cli-babel'],
-        includeExternalHelpers: true,
-        compileModules: false,
-        disableDebugTooling: false,
-        disablePresetEnv: false,
-        disableEmberModulesAPIPolyfill: false,
-      },
-    });
```

Comment:
Can I really remove this? I understand one goal of this task is to remove the dependency of any Ember app to ember-cli-babel, but I am not sure about this part.

In the present work-in-progress I debugged a build to identify what fields ember-cli-babel generates for the app-template and I moved the content from `.embroider/_babel_config_.json` to `babel.config.cjs`.

This 👆 is a set of "forced" options for ember-cli-babel, but it's not _all_ the [available options](https://www.npmjs.com/package/ember-cli-babel#options). What if more options are passed that impact the result list of plugins and presets? Should I keep `.embroider/_babel_config_.json` as it is even if duplicates some plugins with the blueprint? or is this point irrelevant and why?